src/devtools/WebAudioGraphIntegrator.ts

```diff
@@ -635,9 +728,13 @@ function ensureContextsExist(
               );
             }
           } else if (WebAudioRealtimeDataReason.isRealtimeOnlyReason(reason)) {
-            // OfflineAudioContexts emit this error if they are still alive.
+            // OfflineAudioContexts emit this error if they are still
+            // alive.
           } else {
-            console.error(`Unexpected error determining if context '${contextId}' is stale with devtools protocol WebAudio.getRealtimeData.
+            '[' +
+              performance.now().toFixed(2) +
+              '] ' +
+              console.debug(`Unexpected error determining if context '${contextId}' is stale with devtools protocol WebAudio.getRealtimeData.
```

Comment:
This looks like a mistake: `console.debug` is a part of the input of `console.error()`?